runner/src/server/forms/cg-test.json

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
Is this file going to be permanent? We should probably call it something other than `cg-test` if so

Reply:
@gstevenson fixed